src/cswinrt/code_writers.h

```diff
@@ -2650,22 +2688,20 @@ remove => %.ErrorsChanged -= value;
                     if (settings.netstandard_compat)
                     {
                         w.write(R"(
-private % AsInternal(InterfaceTag<%> _) => ((Lazy<%>)_lazyInterfaces[typeof(%)]).Value;
+private % AsInternal(InterfaceTag<%> _) => Make_%();
 )",
                             interface_name,
                             interface_name,
-                            interface_abi_name,
-                            interface_name);
+                            bind<write_lazy_interface_type_name>(interface_type));
                     }
                     else if (is_manually_generated_iface(interface_type))
                     {
                         w.write(R"(
-private % AsInternal(InterfaceTag<%> _) =>  ((Lazy<%>)_lazyInterfaces[typeof(%)]).Value;
+private % AsInternal(InterfaceTag<%> _) =>  Make_%();
```

Comment:
Here and above, you want to first check if the variable already has a value and if not, then make it like we do with the other versions of interlockedexchange.

Reply:
Done